web/package.json

```diff
@@ -42,10 +42,13 @@
   "private": true,
   "dependencies": {
     "@martel/audio-file-decoder": "^2.3.15",
+    "@radix-ui/react-select": "^2.1.4",
+    "@radix-ui/react-slot": "^1.1.1",
```

Comment:
I thought we were adding shadcn to get their components, we're adding radix ui select and slot too? (i saw slot is used in button)

Reply:
shadcn is heavily based on radix